### ISSUE TITLE
LOGO NOT RENDERING FIX

### ISSUE THREAD

**Issue by vehbiu** [InvoiceShelf/InvoiceShelf#125]
## **Is your logo not rendering? Run this command:**
```
docker exec -it invoiceshelf bash -c "cd /var/www/html/InvoiceShelf/resources/views/app/pdf && for dir in estimate invoice locale payment reports; do cd \$dir && sed -i 's/{{ \$logo }}/{{ \"data:\"\.mime_content_type(\$logo)\";base64,\"\.base64_encode(file_get_contents(\$logo)) }}/g' *.php && cd ..; done"
```

## Broken down:

First, get into the docker shell
`docker exec -it invoiceshelf bash`

Then, go to the directory with the issue
`cd /var/www/html/InvoiceShelf/resources/views/app/pdf/`

Then, `cd` into all the template folders (estimate, invoice, locale, payment, reports)

Lastly, replace all occurrences of `{{ $logo }}` (the logo template) with the content as a base64 string
```sed -i 's/{{ \$logo }}/{{ "data:"\.mime_content_type\(\$logo\)\.";base64,"\.base64_encode\(file_get_contents\(\$logo\)\) }}/g' *.php```

**Comment by hari-bo** [comment]
I have tried it but now recieving 500 error when requesting the pdf documents.

**Comment by Marthaarman** [comment]
Thanks, this is a great temporary solution @vehbiu

**Comment by MuratDoganer** [comment]
Im getting this error when trying that fix:

`
syntax error, unexpected double-quoted string ";base64,", expecting ")" (View: /var/www/html/InvoiceShelf/resources/views/app/pdf/invoice/invoice3.blade.php)
`

**Comment by gdarko** [comment]
This is fixed in 2.0.0-RC1 and now available in 2.0.0.
